.github/CONTRIBUTING.md

```diff
@@ -564,6 +564,10 @@ for (var/i in reagents)
 
 * The dlls section of tgs3.json is not designed for dlls that are purely `call()()`ed since those handles are closed between world reboots. Only put in dlls that may have to exist between world reboots.
 
+* Do not use icons, or icon_state strings as overlays. Byond (and by nature, our overlay management code) has to convert these to appearances every time they get added or removed from overlays, a costly operation. Instead Use `icon2appearance()` or `icon_state2appearance()`. These are global procs and can be used in static/global variable defines to avoid re-calling them with the same content.
```

Comment:
```suggestion
* Do not use icons, or icon_state strings as overlays. Byond (and by nature, our overlay management code) has to convert these to appearances every time they get added or removed from overlays, a costly operation. Instead, use `icon2appearance()` or `icon_state2appearance()`. These are global procs and can be used in static/global variable defines to avoid re-calling them with the same content.
```